proto/meta.proto

```diff
@@ -208,6 +208,22 @@ message GetClusterInfoRequest {}
 message GetClusterInfoResponse {
   repeated common.WorkerNode worker_nodes = 1;
   repeated TableFragments table_fragments = 2;
+  message KafkaSplit {
```

Comment:
Is there anything we can reuse? cc @tabVersion

Reply:
Usually we use the `ConnectorSplit` in `source.proto` to represent, can we use the code in connector crate here?

Reply:
Updated with `ConnectorSplit` in `source.proto`